karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       'karma-sourcemap-loader'
     ],
 
-    browsers: ['PhantomJS'],
+    browsers: process.env.CI === true ? ['PhantomJS'] : [],
```

Comment:
https://github.com/mavenlink/brainstem-js/pull/112/commits/9f43fe898e74a0541430a06532ba8b504796af6c

I think it's better to not default to Phantom. Like you said, command-line arguments should allow devs to use Phantom if they want.